.maintain/subwasm.py

```diff
@@ -0,0 +1,114 @@
+import os
+import time
+import argparse
+import sys
+import json
+import requests
```

Comment:
`requests` doesn't included in python batteries,  what is version needs? should add `requirements.txt`?